src/patternfly/components/DataToolbar/data-toolbar.scss

```diff
@@ -253,6 +292,10 @@ $pf-c-data-toolbar--spacer-map: build-spacer-map("none", "sm", "md", "lg");
   }
 }
 
+.pf-c-data-toolbar__toggle.pf-m-expanded .pf-c-button {
+  color: var(--pf-c-data-toolbar__toggle--expanded--button--Color);
```

Comment:
could we update the color in the button (.pf-m-plain) by adding a `pf-m-expanded` modifier like we did to the control modifier. seems like its something we would repeat